test/TestProjects/HlcConstants/Generated/HlcConstantsClient.cs

```diff
@@ -39,14 +39,16 @@ internal HlcConstantsClient(ClientDiagnostics clientDiagnostics, HttpPipeline pi
         }
 
         /// <param name="value"> The RoundTripModel to use. </param>
+        /// <param name="optionalStringQuery"> The Literal to use. The default value is AutoRest.CSharp.Output.Models.Types.EnumTypeValue. </param>
+        /// <param name="optionalBooleanQuery"> The Literal to use. The default value is True. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<RoundTripModel>> MixedAsync(RoundTripModel value, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<RoundTripModel>> MixedAsync(RoundTripModel value, StringConstant? optionalStringQuery = null, bool? optionalBooleanQuery = null, CancellationToken cancellationToken = default)
```

Comment:
Is this expected? all those are constant parameters, why were they in signature?

Reply:
OK. it is optional constant parameter. What's the type `StringConstant`?

Reply:
> OK. it is optional constant parameter. What's the type `StringConstant`?

@chunyu3 It's an optional constant defined like below:
```
"x-ms-enum": {
                            "name": "StringConstant",
                            "modelAsString": false
                        }
```